frontend/packages/portal-frontend/src/resources/components/ResourcesPage.tsx

```diff
@@ -52,43 +53,41 @@ export default function ResourcesPage(props: ResourcesPageProps) {
       </div>
 
       <section className={styles.postsNavList}>
-        {subcategories.map((subcategory: any) => {
-          return (
-            <Accordion key={subcategory.id} title={subcategory.title}>
-              <ListGroup style={{ marginBottom: 0, borderRadius: 0 }}>
-                {subcategory.topics.map((topic: any) => {
-                  return (
-                    <Link
-                      key={topic.id}
-                      to={`?subcategory=${subcategory.slug}&topic=${topic.slug}`}
-                      state={{ postHtml: topic }}
-                      style={{ textDecoration: "none" }}
-                    >
-                      <ListGroupItem
-                        className={styles.navPostItem}
-                        style={{ borderRadius: "0px" }}
-                        active={initPost ? initPost.slug === topic.slug : false}
-                      >
-                        {topic.title}
-                      </ListGroupItem>
-                    </Link>
-                  );
-                })}
-              </ListGroup>
-            </Accordion>
-          );
-        })}
+        <PanelGroup>
+          {subcategories.map((subcategory: Subcategory) => {
+            let isDefaultExpandedSubcategory: boolean;
+            if (querySubcategory) {
+              isDefaultExpandedSubcategory =
+                subcategory.slug === querySubcategory;
+            } else {
+              isDefaultExpandedSubcategory = !!subcategory.topics.find(
+                (t: Topic) => t.slug === defaultTopic?.slug
+              );
+            }
```

Comment:
I'm a little confused about what this if/else block is doing. querySubcategory is if a use arrives on this page from a url query param to a specific resource? But if the user doesn't arrive here from a query param, there might also be a default category set to be open on load?

Reply:
Yes exactly! I think maybe the syntax here is confusing for `isDefaultExpandedSubcategory = !!subcategory.topics…` I believe that was the autosuggest from typescript eslint meaning if there is a value found, set to `true` otherwise set to `false`